src/components/RepoItem.vue

```diff
@@ -27,9 +27,9 @@
       </div>
 
       <div v-if="build" class="build">
-        <img :src="avatar" alt="avatar"/>
+        <img v-if="avatar" :src="avatar" alt="avatar"/>
```

Comment:
I would prefer that we still display the avatar of the commit author. We can modify the Drone server to track deploy author and avatar in a future release, and then replace with the avatar if it makes sense.

Reply:
Sounds good, reverted the changes to the avatar!